lite/Dockerfile

```diff
@@ -56,6 +56,25 @@ RUN git clone --branch 2.0.2-2 --depth 1 "$GIT_SERVER/nanox-secure-sdk.git" "$NA
 ENV NANOSP_SDK=/opt/nanosplus-secure-sdk
 RUN git clone --branch 1.0.4 --depth 1 "$GIT_SERVER/nanosplus-secure-sdk.git" "$NANOSP_SDK"
 
+# Unified SDK
+ENV LEDGER_SECURE_SDK=/opt/ledger-secure-sdk
+RUN git clone "$GIT_SERVER/ledger-secure-sdk.git" "$LEDGER_SECURE_SDK"
+
+# Unified SDK API_LEVEL
+ENV API_LEVEL_1_SDK=/opt/ledger-secure-sdk-api-level-1
+ENV API_LEVEL_2_SDK=/opt/ledger-secure-sdk-api-level-2
+RUN git -C "$LEDGER_SECURE_SDK" worktree add "$API_LEVEL_1_SDK" API_LEVEL_1
+RUN git -C "$LEDGER_SECURE_SDK" worktree add "$API_LEVEL_2_SDK" API_LEVEL_2
+
+ENV NANOSP_1_1_0_SDK=$API_LEVEL_1_SDK
+ENV NANOS2_1_1_0_SDK=$API_LEVEL_1_SDK
+ENV NANOX_2_1_0_SDK=$API_LEVEL_1_SDK
```

Comment:
Add versions for existing nanos and unreleased nanox using the `API_LEVEL_0_SDK` / `API_LEVEL_NONE_SDK` mentioned above